dumpling-overview.md

```diff
@@ -65,11 +65,10 @@ In the above commands, you need to modify `~/.bash_profile` to the path of your
 
 ### Required privileges
 
-- SELECT
-- RELOAD
-- LOCK TABLES
-- REPLICATION CLIENT
-- PROCESS
+- SELECT: Required when exporting tables.
+- RELOAD: Required when using `consistency flush`. Note that only TiDB supports this privilege. When the upstream is an RDS database or a managed service, you can ignore this privilege.
+- LOCK TABLES: Required when using `consistency lock`. This privilege must be granted for all the databases and tables to be exported.
+- REPLICATION CLIENT: Required when exporting metadata to record data snapshot. This privilege is optional.
```

Comment:
```suggestion
- REPLICATION CLIENT: Required when exporting metadata to record data snapshot. This privilege is optional if you do not need to export metadata.
```

Reply:
IMO, optional is a fixed attribute and does not change if the user wants to export metadata. How about:
> This privilege is optional and you can ignore it if you do not need to export metadata.

Reply:
LGTM